src/components/EventBox/Request-view.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+// Figure out what inputs we want
+type RequestProps = {
```

Comment:
This is for displaying repair requests (issue 118)

Reply:
Originally, we thought we were supposed to create the page to view repair requests ourselves, but we looked at issue 118 which seemed to be exactly that. We decided not to go any further to avoid messing with other issues. We left it in for testing - should be replaced by issue 118 once both issues are merged, if we are understanding the project structure correctly.